neo/Prompt/Commands/Wallet.py

```diff
@@ -305,6 +306,66 @@ def command_desc(self):
         return CommandDesc('unspent', 'show unspent assets', params=[p1, p2, p3, p4])
 
 
+class CommandWalletSplit(CommandBase):
+
+    def __init__(self):
+        super().__init__()
+
+    def execute(self, arguments):
+        wallet = PromptData.Wallet
+
+        if len(arguments) < 4:
+            print("Please specify the required parameters")
+            return None
+
+        if len(arguments) > 5:
+            # the 5th argument is the optional attributes,
+            print("Too many parameters supplied. Please check your command")
+            return None
+
+        try:
+            from_addr = wallet.ToScriptHash(arguments[0])
+        except ValueError as e:
+            print(str(e))
+            return None
+
+        asset_id = PromptUtils.get_asset_id(wallet, arguments[1])
```

Comment:
I think we should test for `asset_id` being `None` here and give a user friendly error message that the asset is bad. Right now it gives the following:
```
neo> wallet split AK2nJJpJr6o664CWJKi1QRXjqeic2zRp8y bla 0 2
No unspent assets matching the arguments.
```
This initially had me thinking that it was failing on a wrong `unspent_index`.